protos/services.proto

```diff
@@ -53,4 +53,19 @@ message ValidateFramePointersResponse {
 service FramePointerValidatorService {
   rpc ValidateFramePointers(ValidateFramePointersRequest)
       returns (ValidateFramePointersResponse) {}
-}
\ No newline at end of file
+}
+
+message GetCrashRequest {
```

Comment:
Hi @IrinaShkviro , could you rename these to `CrashOrbitServiceRequest`, `CrashOrbitServiceResponse` to follow the convention of `<function_name>`{`Request`,`Response}`?

Reply:
Yes, sure. Thank you!

Reply:
Sorry for the delay, created a PR #742